contrib/spec/podman.spec.in

```diff
@@ -64,7 +64,9 @@ Requires: containers-common
 Requires: containernetworking-cni >= 0.6.0-3
 Requires: iptables
 Requires: oci-systemd-hook
+%if 0%{?fedora}
 Recommends: container-selinux
+%endif
```

Comment:
Should have %else
Requires: container-selinux

Reply:
fixed